calicoctl/calico_ctl/diags.py

```diff
@@ -135,9 +145,11 @@ def save_diags(log_dir):
             f.writelines("Error running ipset: %s\n" % e)
 
     # Ask Felix to dump stats to its log file - ignore errors as the
-    # calico-node might not be running
-    subprocess.call(["docker", "exec", "calico-node",
-                     "pkill", "-SIGUSR1", "felix"])
+    # calico-node might not be running.  If using rkt runtime, skip this step, 
+    # since we can't yet run commands inside a rkt fly pod.
+    if runtime == "docker":
+        subprocess.call(["docker", "exec", "calico-node",
+                         "pkill", "-SIGUSR1", "felix"])
 
     if os.path.isdir(log_dir):
         print("Copying Calico logs")
```

Comment:
When running under rkt, the calico-node logs will be in the journal and not on the filesystem. Should we call out to the journal instead?